docs/source/installation.rst

```diff
@@ -76,6 +81,120 @@ If testing or docs is desired, additional steps are required.
     (linchpin) $ pip install linchpin[docs]
     (linchpin) $ pip install linchpin[tests]
 
+
+Installing LinchPin on Fedora 26
+---------------------------------
+
+Install RPM pre-reqs
+
+.. code-block:: bash
+
+    $ dnf -y install python-virtualenv libffi-devel openssl-devel libyaml-devel gmp-devel libselinux-python make gcc redhat-rpm-config libxml2-python
+
+
+Create a working-directory
+
+.. code-block:: bash
+
+    $ mkdir mywork
+    $ cd mywork
+
+Create linchpin directory, make a virtual environment, activate the virtual environment
+
+.. code-block:: bash
+
+    $ mkdir linchpin
+    $ virtualenv --system-site-packages linchpin
+    $ source linchpin/bin/activate
+    (linchpin) $
+
+Install linchpin
+
+.. code-block:: bash
+
+    (linchpin) $ pip install linchpin
+
+Make a workspace, and initialize it to prove that linchpin itself works
+
+.. code-block:: bash
+
+    (linchpin) $ mkdir workspace
+    (linchpin) $ export WORKSPACE=./workspace
+    (linchpin) $ linchpin init
+    PinFile and file structure created at /root/work/workspace
```

Comment:
Could this not assume root? Maybe make it say something like `/home/user/workspace`?